### PR TITLE
changes to CareConnect VaccinationProcedure Valueset

### PR DESCRIPTION
**Pull request by SBajanki1**
To add SNOMED codes to CareConnect Vaccination Procedure ValueSet, change the valueset binding to extensible in Extension-CareConnect-VaccinationProcedure-1.xml

### CODE REVIEW

File: structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml [hunk 1]
```diff
@@ -28,195 +28,6 @@
   <type value="Extension" />
   <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
   <derivation value="constraint" />
-  <snapshot>
```
Comment (APageNHS) [inline structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml:31]: I'm guessing the snapshot has been removed due to Forge. Given the other extensions still have snapshot, we possibly want to ensure this is still present here.

File: structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml [hunk 2]
```diff
@@ -237,7 +48,7 @@
         <code value="CodeableConcept" />
       </type>
       <binding>
-        <strength value="required" />
+        <strength value="extensible" />
```
Comment (APageNHS) [inline structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml]: In UKCore, we've gone with Preferred as the strength for SNOMED CT valuesets - might be worth considering that in CareConnect?

File: structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml [hunk 3]
```diff
@@ -228,7 +39,7 @@
       <path value="Extension.url" />
       <fixedUri value="https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-VaccinationProcedure-1" />
     </element>
-    <element id="Extension.valueCodeableConcept:valueCodeableConcept">
+    <element id="Extension.value[x]:valueCodeableConcept">
```
Comment (APageNHS) [inline structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml:43]: Presumably won't be a breaking change

File: structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml (hunk 2 above)
Comment (SBajanki1) [reply structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml]: Thank you, I'll have a look a make changes

File: structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml (hunk 1 above)
Comment (SBajanki1) [reply structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml:31]: Looked at the UKCore page (https://simplifier.net/guide/UK-Core-Implementation-Guide-STU3-Sequence/Home/ProfilesandExtensions/ExtensionLibrary/Extension-UKCore-VaccinationProcedure.page.md?version=current) , <snapshot> is not there, so didn't include that

Comment (APageNHS) [reply structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml:31]: I meant the other CareConnect extensions - have checked with Marcus who says we should keep the snapshots in CareConnect - this should just be a setting in your Forge (and I think FHIR STU3 uses a different version of Forge?)

**Pull request closed without merging**